src/bmm-skills/plan/bmad-sprint-planning/scripts/sprint_plan.py

```
@@ -0,0 +1,446 @@
+        print(json.dumps(result))
+        return
+
+    _atomic_write(yaml, doc, args.status_file)
+    try:
+        verify_yaml = YAML()
+        with io.open(args.status_file, "r", encoding="utf-8") as fh:
+            reread = verify_yaml.load(fh)
+        assert dict(reread["development_status"]) == dict(dev), "development_status mismatch after write"
+        for field in ("generated", "last_updated", "project"):
+            assert reread[field] == doc[field], f"{field} mismatch after write"
+    except BaseException as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)